app/code/Riskified/Decider/Api/Order.php

```diff
@@ -197,6 +201,10 @@ private function load($model)
         $order->line_items = $this->_orderHelper->getLineItems();
         $order->shipping_lines = $this->_orderHelper->getShippingLines();
 
+        if(empty($this->_orderHelper->getRefundDetails()) != 1){
```

Comment:
We are running same method twice. Please assign result of the method to variable and work on it.